test/ibmq/test_ibmq_job.py

```diff
@@ -400,16 +401,16 @@ def test_retrieve_jobs_start_datetime(self, provider):
         """Test retrieving jobs created after a specified datetime."""
         backend = provider.get_backend('ibmq_qasm_simulator')
         past_month = datetime.now() - timedelta(days=30)
-        past_month_str = past_month.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+        past_month = past_month.replace(tzinfo=tz.tzlocal())
```

Comment:
Why did you add `tzinfo` to all these test cases? The users are more likely to pass in a datetime without `tzinfo`.

Reply:
I added `tzinfo` since it was needed for comparison against the date returned by `creation_date` (which is tz aware).

Since users are most likely to use non timezone aware date times, I'll search for the jobs with a non timezone aware datetime and compare using the datetime with `tzinfo`.